src/ndarray.jl

```diff
@@ -1033,31 +1035,53 @@ function _get_function_expressions(handle :: MX_handle, name)
   if name == :dot
     _use_vars.args[2:end] = flipdim(_use_vars.args[2:end], 1)
   end
+
+  if name == :transpose
+    transform = quote
+      new_kwargs = Any[]
+      for (key, arg) in kwargs
```

Comment:
This is minor, but maybe using a list comprehension expression instead of for-loop + `push!` might be more efficient?

Could you also fix it for the symbolic version?